playbooks/siege.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Install siege on ansible host
+  hosts: ansiHost
+  become: yes
+
+  tasks:
+    - include_role:
+        name: siege-install
+      tags:
+        - siege
+
+    # - include_role:
+    #     name: siege-run
+    #   tags:
+    #     - siege
```

Comment:
Just curious, is this commented out intentionally?

Reply:
Yes. This will end up being part of the sec playbook.